index.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 New Relic, Inc.  Licensed under the Apache License, version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND, either express or implied. 
+//Copyright 2018 New Relic Inc. All rights reserved.
+
+import fetch from 'node-fetch';
+import { ApiClient, RealtimeApi } from "fastly";
+
+const ACCOUNT_ID = process.env.ACCOUNT_ID
+const INSERT_KEY = process.env.INSERT_KEY
+const FASTLY_KEY = process.env.FASTLY_KEY
+const INSIGHTS_HOST = process.env.INSIGHTS_HOST || "insights-collector.newrelic.com";
+const EVENT_TYPE = process.env.EVENT_TYPE || "LogAggregate";;
+const POLL_INTERVAL = process.env.POLL_INTERVAL || 180000 // 3 minutes.
+
+// A limitation of Fastly is that you have to query one service at a time. I chose to create an array of my service ids, and loop through them.
+const LIST_OF_SERVICES = process.env.SERVICES.split(" ");
+
+let timestamp = 0;
+
+const fastlyApi = new ApiClient();
```

Comment:
This is a temporary change until Fastly can address an endpoint [problem](https://github.com/fastly/fastly-js/issues/18) in their API client.